PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -111,10 +108,18 @@ def _tauId8WPMask(pattern,doc):
     idMVAoldDM2017v2 = _tauId7WPMask( "by%sIsolationMVArun2v1DBoldDMwLT2017v2", doc="IsolationMVArun2v1DBoldDMwLT ID working point (2017v2)"),
     idMVAoldDMdR032017v2 = _tauId7WPMask( "by%sIsolationMVArun2v1DBdR03oldDMwLT2017v2", doc="IsolationMVArun2v1DBoldDMdR0p3wLT ID working point (2017v2)")
 )
-_mvaAntiEVars2018 = cms.PSet(
+_mvaAntiEVars = cms.PSet(
        rawAntiEle2018 = Var("tauID('againstElectronMVA6Raw2018')", float, doc= "Anti-electron MVA discriminator V6 raw output discriminator (2018)", precision=10),
        rawAntiEleCat2018 = Var("tauID('againstElectronMVA6category2018')", int, doc="Anti-electron MVA discriminator V6 category (2018)"),
        idAntiEle2018 = _tauId5WPMask("againstElectron%sMVA62018", doc= "Anti-electron MVA discriminator V6 (2018)"),
+       rawAntiEle = Var("tauID('againstElectronMVA6Raw')", float, doc= "Anti-electron MVA discriminator V6 raw output discriminator (2015)", precision=10),
+       rawAntiEleCat = Var("tauID('againstElectronMVA6category')", int, doc="Anti-electron MVA discriminator V6 category (2015)"),
+       idAntiEle = _tauId5WPMask("againstElectron%sMVA6", doc= "Anti-electron MVA discriminator V6 (2015)"),
+)
+_mvaAntiEVars2015 = cms.PSet(
+       rawAntiEle = Var("tauID('againstElectronMVA6Raw')", float, doc= "Anti-electron MVA discriminator V6 raw output discriminator (2015)", precision=10),
+       rawAntiEleCat = Var("tauID('againstElectronMVA6category')", int, doc="Anti-electron MVA discriminator V6 category (2015)"),
+       idAntiEle = _tauId5WPMask("againstElectron%sMVA6", doc= "Anti-electron MVA discriminator V6 (2015)")
```

Comment:
this is not 1:1 backport
the labelling of 2015 and 2018 is done differently between here and master.
is this intended ?

Reply:
I am surprised as I think it is OK, but let me check - a bug is not excluded. I will back to it.

Reply:
I checked it again and it is correct as it is. Difference wrt master is caused by the fact that here in "default workflows", i.e. other than `run2_miniAOD_devel` it is expected that anti-e tauIDs 2015 are not labeled by year in patTaus, while 2018 ones are rerun as a part of nanoAOD sequences and labelled by 2018 in patTaus. The order is swapped in case of `run2_miniAOD_devel` with 106X as well as in default workflow with master.